health/mapping/es.py

```diff
@@ -76,11 +77,31 @@
 
 
 def init_elastic(es, index_to_create="ms_health_idx_1"):
-    r = requests.get("%s/%s" % (es, "ms_health"))
-    if r.status_code == 200:
-        print("Index ms_health already exists, nothing to do!")
+    # TODO(kzaitsev): replace prints with logging
+    r = requests.put("%s/%s" % (es, index_to_create),
```

Comment:
Actually this leaves me with Got 400 status when creating index 'ms_health_idx_1' and index_already_exists_exception